package.json

```
@@ -20,6 +21,12 @@
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
   "scripts": {
+    "build:all": "run-p build:11ty build:css",
+    "dev:11ty": "yarn dev && yarn build:11ty && NODE_ENV=development BROWSERSYNC_OPEN=true npx @11ty/eleventy --config=./site/.eleventy.cjs --serve",
+    "build:11ty": "npx @11ty/eleventy --config=./site/.eleventy.cjs",
+    "build:css": "node ./tasks/build-css.js && pretty-quick --fix --pattern \"{components}/**/*.css\"",
+    "build:css:watch": "node ./tasks/watch-css.js",
+    "build:watch": "run-p build:css:watch",
```

```suggestion
    "dev:docs": "NODE_ENV=development BROWSERSYNC_OPEN=true yarn workspace spectrum-site run build --serve",
    "build:docs": "yarn workspace spectrum-site run build",
```

The site folder should be treated as a workspace and have it's own package with commands sourced from there in order to encapsulate dependencies.

---

Done.
@castastrophe Created a new folder called content and moved the nunjucks and md files inside them. You might see some files deleted and added in the commits. :)